README.md

```diff
@@ -27,7 +27,11 @@ You can retrieve binaries for Linux, macOS and Windows on our [GitHub release pa
 ##### Linux
 If you prefer to do it straight from CLI here's a one-liner for you:
 ```bash
-curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
+curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJ ./opencompose
```

Comment:
@cdrage It should not!

Then you would have to specify a file to work with to `tar`. The correct way with shell would be:
```bash
set -o pipefail && curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJ ./opencompose
```
but I am not putting that in installation instructions. The reason is that this command is meant for interactive use and in case `curl` fails user will see an error from both `curl` and `tar` and it will also give non-zero return code even without the `set -o pipefail` option because if `curl` won't download an archive, `tar` will fail because it has corrupted or missing input.